addons/website_blog/data/website_blog_demo.xml

```diff
@@ -329,5 +336,68 @@ Naturally, as you grow in your love of astronomy, you will find many celestial b
 ]]>
             </field>
         </record>
+
+        <record id="blog_comment_1" model="mail.message">
+            <field name="body">Beautiful !, I know where will be my next trip !</field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_1"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+        </record>
+
+        <record id="blog_comment_2" model="mail.message">
+            <field name="body">Hi ! How much time did you went there ? </field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_1"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_demo"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+        </record>
+
+        <record id="blog_comment_3" model="mail.message">
+            <field name="body">I'll follow your instructions next time ! It will save myself from weird place like last time :D</field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_3"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+        </record>
+
+        <record id="blog_comment_4" model="mail.message">
+            <field name="body">Can't wait to buy a telescope !</field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_5"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_demo"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+        </record>
+
+        <record id="blog_comment_5" model="mail.message">
+            <field name="body">Light pollution can be really annoying sometimes. I like to go camping to be able to see our amazing sky !</field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_5"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
```

Comment:
Change author: Joel Willis
"Light pollution can be really annoying when you want to observe the sky. The best places are far from cities. That's why I like to go camping."